setup.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import find_packages, setup
+
+from opencensus_python_extensions_azure import __version__
+
+setup(
+    name='opencensus-python-extensions-azure',
+    version=__version__,  # noqa
+    author='OpenCensus Azure',
+    author_email='opencensusazure@microsoft.com',
```

Comment:
I don't believe this alias exists. Either use your teams DG or app insights team.

Reply:
I registered it recently. Just sent out a test email.